### ISSUE TITLE
Support for namespace classes

### ISSUE THREAD

**Issue by alex-zywicki** [dedoussis/asynction#147]
[Class-Based Namesaces](https://flask-socketio.readthedocs.io/en/latest/getting_started.html#class-based-namespaces) Would be a nice feature to have. Validation and security may be a little tricky to get the bindings wrapped on the correct class functions, but wouldn't be too hard

**Comment by dedoussis** [comment]
Agreed. This is something that Asynction should support. Will add it on the roadmap.
